wp-content/themes/pub/wporg-learn-2024/src/style/_sensei.scss

```diff
@@ -80,3 +80,36 @@ body.sensei {
 	margin: unset;
 	padding: unset;
 }
+
+.wporg-learn-sidebar-meta-info {
```

Comment:
On second thoughts, I think all of this should be moved to a `_sidebar.scss` file, but it's not blocking.

Reply:
725db9a